tests/integration/test_helm.py

```diff
@@ -0,0 +1,30 @@
+import subprocess
+import time
+
+def test_helm_chart_deployment():
+    # Set the Helm chart name and release name
+    chart_name = "cdf-fabric-replicator-chart"
+    release_name = "int-test-helm"
+
+    subprocess.run("pwd", shell=True, check=True)
+
+    # Install the Helm chart
+    install_command = f"helm install {release_name} {chart_name}"
+    subprocess.run(install_command, shell=True, check=True)
+
+    # Check if the release is deployed successfully
+    status_command = f"helm status {release_name}"
+    status_output = subprocess.check_output(status_command, shell=True).decode("utf-8")
+    assert "STATUS: deployed" in status_output, "Helm chart deployment failed"
+
+    # Add a 30-second sleep to allow time for the pods to start
+    time.sleep(30)
```

Comment:
We have a good pattern for more deterministic behaviors instead of using sleep. Take a look at https://github.com/cognitedata/cdf-fabric-replicator/blob/10417399218c53a9b719d761f8839899edf025d4/tests/integration/integration_steps/cdf_steps.py#L319 for an example

Reply:
Added retry logic